doc/ref_arch/openstack/chapters/chapter02.md

```diff
@@ -55,95 +55,108 @@ To ensure alignment with the infrastructure profile catalogue, the following req
+| infra.net.cfg.003 | Network Address Translation | Must support | Must support | |
+| infra.net.cfg.004 | Security Groups | Must support | Must support | |
+| infra.net.cfg.005 | SFC support | Not required | Must support | |
+| infra.net.cfg.006 | Traffic patterns symmetry | Must support | Must support | |
+
+<p align="center"><b>Table 2-2a:</b> Reference Model Requirements: Virtual Networking</p>
+
+> * [Workload Transition Guidelines](../chapters/appendix-a.md) might have other interfaces (such as SR-IOV VFs to be directly passed to a VM) or NIC-specific drivers on guest machines transiently allowed until more mature solutions are available with an acceptable level of efficiency to support telecom workloads (for example regarding CPU and energy consumption).
 
-The required number of connection points to a VM is described in `e.cap.004` above.  This section describes the required bandwidth of those connection points.
+The required number of connection points to a VM is described in `e.cap.004` [above](#2.2.1).  The table below specifies the required bandwidth of those connection points.
 
-| Reference  | Description | Requirement for Basic Profile | Requirement for Network Intensive Profile| Specification Reference |
+| Reference  | Description | Requirement for Basic Profile | Requirement for High Performance Profile| Specification Reference |
 |---|---|---|---|---|
 | n1, n2, n3, n4, n5, n6	| 1, 2, 3, 4, 5, 6 Gbps | Must support | Must support | |
 | n10, n20, n30, n40, n50, n60	| 10, 20, 30, 40, 50, 60 Gbps | Must support | Must support | |
-| n25, n50, n75, n100, n125, n150	| 25, 50, 75, 100, 125, 150 Gbps | Must support | Must support | |
-| n50, n100, n150, n200, n250, n300	| 50, 100, 150, 200, 250, 300 Gbps | Must support | Must support | |
-| n100, n200, n300, n400, n500, n600	| 100, 200, 300, 400, 500, 600 Gbps | Must support | Must support | |2.2](../../../ref_model/chapters/chapter04.md#422-virtual-network-interface-specifications) | n1, n2, n3, n4, n5, n6	| 1, 2, 3, 4, 5, 6 Gbps | Must support | Must support | |
+| n25, n50, n75, n100, n125, n150	| 25, 50, 75, 100, 125, 150 Gbps | Optional | Must support | |
+| n50, n100, n150, n200, n250, n300	| 50, 100, 150, 200, 250, 300 Gbps | Optional | Must support | |
+| n100, n200, n300, n400, n500, n600	| 100, 200, 300, 400, 500, 600 Gbps | Optional | Must support | |
 
+<p align="center"><b>Table 2-2b:</b> Reference Model Requirements: Network Interface Specifications</p>
+
+<a name="2.2.2.1"></a>
+### 2.2.2.1 Cloud Infrastructure Software Profile Extensions Requirements for Networking
+
+| Reference  | Description | Requirement for Basic Profile | Requirement for High Performance Profile| Specification Reference |
+|---|---|---|---|---|
+| infra.net.acc.cfg.001 |	vSwitch optimisation |	N |	Y, DPDK | |
```

Comment:
@pgoyal01 I think DPDK should be part of the descritption 'vSwitch optimisation with DPDK'
SR-IOV is no more mentioned in profile requirement and neither in Profile extension.  
It should be better located in 2.2.2.1 'Profile Extension', as we can build a High performance node without SR-IOV in case of compute intensive node for instance.
It means that it should be changed consequently in RM

Reply:
@karinesevilla Thanks. Made changes:
- in 2.2.2.1 changed the entry for infra.net.acc.cfg.001 and infra.hw.nac.cfg.004 (SR-IOV) to align with RM 4.2.2; added references to RM
- in 2.4.2, changed `req.inf.acc.03` (SR-IOV) from "should not" to "may"